### PR TITLE
add: populate secure comm overview

### PR DESCRIPTION
**Pull request by DannyS03**
## Context

- Adds overview for secure communication 
- Part of https://github.com/libp2p/docs/issues/19

### Notes

Section this document lives under will become populated with corresponding PRs.

## Latest preview

Please view the latest Fleek preview [here](https://bafybeie4vcl5j5hgmg4ejsp5eqkdrzkhw4cdtu7x4w3zbszhylrndabbfm.on.fleek.co/concepts/secure-comm/overview/).

### CODE REVIEW

File: content/concepts/secure-comm/overview.md [hunk 1]
```diff
@@ -8,8 +8,16 @@ aliases:
...
-Before two peers can transmit data, the communication channel they established
-with a transport protocol should be secure. A transport protocol like QUIC provides
-security guarantees out-of-the-box, but other transports in libp2p do not provide
-the logic to secure their channel. This requires an upgrade to the transport using an
-upgrader. Security is always established first over the raw connection.
+Before two peers can transmit data, the communication channel they
+establish needs to be secured. By design,
+libp2p supports many different transports (TCP, QUIC, WebSocket, WebTransport,
+etc.). Some transports have built-in encryption at the transport layer (e.g. QUIC)
+like [QUIC](../transports/quic), while other transports (e.g. TCP, WebSocket)
```
Comment (marten-seemann) [inline content/concepts/secure-comm/overview.md:15]: QUIC is mentioned twice here.

File: content/concepts/secure-comm/overview.md [hunk 2]
```diff
@@ -8,8 +8,16 @@ aliases:
...
+lack native security and require a security handshake after the connection has been
+established.
+
+## Secure channels in libp2p
+
+Libp2p supports two security protocols, [TLS 1.3](tls) and [Noise](noise).
```
Comment (marten-seemann) [inline content/concepts/secure-comm/overview.md:21]:
```suggestion
libp2p specifies two security protocols, [TLS 1.3](tls) and [Noise](noise).
```

File: content/concepts/secure-comm/overview.md [hunk 3]
```diff
@@ -8,8 +8,16 @@ aliases:
...
-with a transport protocol should be secure. A transport protocol like QUIC provides
-security guarantees out-of-the-box, but other transports in libp2p do not provide
-the logic to secure their channel. This requires an upgrade to the transport using an
-upgrader. Security is always established first over the raw connection.
+Before two peers can transmit data, the communication channel they
+establish needs to be secured. By design,
+libp2p supports many different transports (TCP, QUIC, WebSocket, WebTransport,
+etc.). Some transports have built-in encryption at the transport layer (e.g. QUIC)
+like [QUIC](../transports/quic), while other transports (e.g. TCP, WebSocket)
+lack native security and require a security handshake after the connection has been
```
Comment (marten-seemann) [inline content/concepts/secure-comm/overview.md:16]:
```suggestion
lack native security and require a security handshake after the transport connection has been
```

**Review by marten-seemann — APPROVED** [review]

File: content/concepts/secure-comm/overview.md [hunk 4]
```diff
@@ -8,8 +8,16 @@ aliases:
...
 
-Before two peers can transmit data, the communication channel they established
-with a transport protocol should be secure. A transport protocol like QUIC provides
-security guarantees out-of-the-box, but other transports in libp2p do not provide
-the logic to secure their channel. This requires an upgrade to the transport using an
-upgrader. Security is always established first over the raw connection.
+Before two peers can transmit data, the communication channel they
+establish needs to be secured. By design,
+libp2p supports many different transports (TCP, QUIC, WebSocket, WebTransport,
+etc.). Some transports have built-in encryption at the transport layer (e.g. QUIC)
```
Comment (DannyS03) [inline content/concepts/secure-comm/overview.md:14]:
```suggestion
etc.). Some transports have built-in encryption at the transport layer
```

**Pull request merged**